che-theia-task-extension/package.json

```diff
@@ -29,7 +29,8 @@
     "@theia/core": "0.3.14",
     "@theia/mini-browser": "0.3.14",
     "@theia/task": "0.3.14",
-    "@eclipse-che/workspace-client": "0.0.1-1546444839",
+    "@eclipse-che/workspace-client": "latest",
+    "@eclipse-che/api": "latest",
```

Comment:
are we allowed to use non fixed version in Eclipse projects ? @benoitf

Reply:
But all of them is Eclipse projects, why not?

Reply:
OK :)